Header header
sensor_msgs/NavSatFix GNSSpose
sensor_msgs/Image Image